docs/api/options.md

```diff
@@ -323,7 +323,18 @@ wrapper.destroy()
 - default: `false`
 
 ::: warning Deprecation warning
-`attachToDocument` is deprecated and will be removed in future releases. Use [`attachTo`](#attachto) instead.
+`attachToDocument` is deprecated and will be removed in future releases. Use [`attachTo`](#attachto) instead. Example:
```

Comment:
Could we add a very simple explanation of what does the example do? Say, "For example, if you need to attach the component to the `document.body`" or similar